docs/install.rst

```diff
@@ -184,6 +184,30 @@ globally.
    available globally. This helps avoid issues caused by hiding the public
    schema from queries.
 
+Working with Tenant specific schemas
+====================================
+Since each Tenant has it's own database you need a way to tell Django what
+database to use when using the management commands. 
+
+A special management command ``tenant_command`` has been added to allow you to
+execute Django management commands on a specific Tenant schema.
```

Comment:
It's not certain what schema a management command will operate on.

If you have a management command that modifies data in a shared application, you could easily be giving readers the impression that it won't be having an effect on other tenants.

It is probably better to indicate that the context of the management command will be switched to that of the tenant, and then explicitly give a warning that depending on operation it may well be impacting on shared data too.